### PR TITLE
add vuex-persist

### PR DESCRIPTION
**Pull request by championswimmer**

### CODE REVIEW

**Comment by championswimmer** [comment]
@kazupon done !

File: README.md [hunk 1]
```diff
@@ -1193,6 +1193,7 @@ Tooltips / popovers
  - [vuex-action](https://github.com/varHarrie/vuex-action) - Utilities for vuex to easily create and manage actions.
  - [vuex-rest-api](https://github.com/christianmalek/vuex-rest-api) - A Helper utility to simplify the usage of REST APIs with Vuex. Based on axios.
  - [vuex-cache](https://github.com/superwf/vuex-cache) - A Vuex plugin utility to cache action those will make remote request.
+ - [vuex-persist](https://github.com/championswimmer/vuex-persist) - A Typescript-ready Vuex plugin to help save the store to localStorage or any custom Storage (that you can configure)
```
Comment (nickmessing) [inline README.md]: @championswimmer, can you pelase add a `.` (dot) at the end? Just to be consistent.

**Comment by championswimmer** [comment]
@nickmessing  done !

Please squash and merge, as it has become 3 commits now.

**Comment by nickmessing** [comment]
@championswimmer, that's what we always do ;)

**Comment by nickmessing** [comment]
@kazupon, any objections?

**Pull request merged**